indico/modules/events/editing/models/file_types.py

```diff
@@ -55,6 +55,11 @@ def __table_args__(cls):
         nullable=False,
         default=False
     )
+    filename_template = db.Column(
+        db.String,
+        nullable=True,
+        default=''
```

Comment:
That's contradicting with `nullable=True`: We should decide whether we want to use an empty string or None for "no template". If we go for None, there shouldn't be a default (implicitly defaults to None/NULL); otherwise it should be `nullable=False`

Reply:
In the past (WTForms) we usually used empty strings with `nullable=False` because this was more straightforward - zero extra steps necessary, and no risk of getting a mix of the two values.

With final-form and webargs/marshmallow the overhead for empty-to-null is minimal, but of course there's still the risk of getting empty strings in the DB if we don't validate properly (unless we add a CHECK, and personally I'd prefer to avoid creating CHECKs for every nullable string field).

cc @indico/development-team

Reply:
Yeah, you are right. This shouldn't have any default value.

> With final-form and webargs/marshmallow the overhead for empty-to-null is minimal

Yep, that sounds like a good reason to stop providing defaults on the SQLAlchemy level.